lib/Jsrt/ChakraCommon.h

```diff
@@ -1588,7 +1926,7 @@
     /// <param name="data">
     ///     The external data that was passed in when creating the object being finalized.
     /// </param>
-    typedef void (CALLBACK *JsFinalizeCallback)(_In_opt_ void *data);
+    typedef void (CHAKRA_CALLBACK *JsFinalizeCallback)(_In_opt_ void *data);
```

Comment:
typedef enum _JsPropertyIdType
typedef enum _JsValueType
CHAKRA_CALLBACK *JsFinalizeCallback
CHAKRA_CALLBACK * JsNativeFunction
CHAKRA_CALLBACK *JsPromiseContinuationCallback

should be moved to top before any function deceleration to be consistent